plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/WidgetHandler.java

```diff
@@ -104,6 +104,9 @@ public Boolean run() {
 	 * @return true if widget is visible, false otherwise
 	 */
 	public boolean isVisible(Widget widget) {
+		if (widget.isDisposed())
```

Comment:
Roll back those changes in handler. We do not check whether a widget is disposed in handler methods.